zio-http/src/main/scala/zio/http/ClientSSLConfig.scala

```diff
@@ -24,4 +26,20 @@ object ClientSSLConfig {
   final case class FromCertResource(certPath: String)                                         extends ClientSSLConfig
   final case class FromTrustStoreResource(trustStorePath: String, trustStorePassword: String) extends ClientSSLConfig
   final case class FromTrustStoreFile(trustStorePath: String, trustStorePassword: String)     extends ClientSSLConfig
+
+  lazy val config: Config[ClientSSLConfig] = {
```

Comment:
Maybe incorporate @adamgfraser's work defining this one using the new machinery?

Reply:
we can merge it on top of it